src/azul/plugins/metadata/hca/contributor_matrices.py

```diff
@@ -26,6 +27,61 @@
 ]
 
 
+def parse_stratification(strata: str) -> JSONs:
```

Comment:
Looking at the call sites

```suggestion
def parse_strata(strata: str) -> JSONs:
```